python/lib/scheduler/dmod/scheduler/job/job_manager.py

```diff
@@ -673,43 +683,52 @@ async def manage_job_processing(self):
             organized_lists = self._organize_active_jobs(active_jobs)
             jobs_eligible_for_allocate = organized_lists[0]
             jobs_to_release_resources = organized_lists[1]
-            jobs_completed_phase = organized_lists[2]
 
             for job_with_allocations_to_release in jobs_to_release_resources:
                 result = self.release_allocations(job_with_allocations_to_release.job_id)
                 if not result.success:
                     logging.error(f"Failure deallocating {job_with_allocations_to_release.job_id} due to "
                                   f"'{result.reason}' (message was: `{result.message}`)")
 
-            for job_transitioning_phases in jobs_completed_phase:
-                # TODO: figure out what to do here; e.g., start output service after model_exec is done
-                pass
+            # TODO: figure out what to do here; e.g., start output service after model_exec is done
+            #jobs_completed_phase = organized_lists[2]
```

Comment:
I would just remove these lines instead of commenting them out. Its up to your discretion.

Reply:
I only left it there because it seemed like there was a root of an idea there. Do I know what it was? No clue, but there _is_ a hint of an intention, so I wasn't sure whether or not to remove it. I'm for leaving it at the moment, just so people can see it if/when they go to play with it later and understand "Hey, this was designed with the intention that some step occurs here"